ocsp/Cargo.toml

```
@@ -0,0 +1,30 @@
+[package]
+name = "ocsp"
```

The `ocsp` crate name is taken, however you can use this:
```suggestion
name = "x509-ocsp"
```

---

Changed.